product_warranty/views/product_warranty.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- return instructions tree view -->
+    <record id="product_return_instructions_tree_view" model="ir.ui.view">
+        <field name="name">product.return.instructions.tree</field>
+        <field name="model">return.instruction</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="instructions" />
+                <field name="is_default" />
+            </tree>
+        </field>
+    </record>
+    <!-- return instructions form view -->
+    <record id="product_return_instructions_form_view" model="ir.ui.view">
+        <field name="name">product.return.instructions.form</field>
+        <field name="model">return.instruction</field>
+        <field name="arch" type="xml">
+            <form string="Return instructions">
+                <sheet>
+                    <group name="return_instructions" col='4'>
+                        <field name="name" />
+                        <field name="is_default" />
+                    </group>
+                    <separator string="Instructions" colspan="4" />
+                    <field name="instructions" nolabel="1" colspan="4" />
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="product_return_instructions_action" model="ir.actions.act_window">
+        <field name="name">Products Return Instructions</field>
+        <field name="res_model">return.instruction</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="product_return_instructions_tree_view" />
+    </record>
+    <menuitem
+        action="product_return_instructions_action"
+        groups="base.group_no_one"
+        id="menu_product_return_instructions_action"
```

Comment:
Please move the instructions to their own `views/product_return_instructions_views.xml` file.

Reply:
Done!